did/plugins/jira.py

```diff
@@ -425,6 +452,42 @@ def fetch(self):
             query = query + f" AND project in ({self.parent.project})"
         self.stats = Issue.search(query, stats=self)
 
+
+class JiraWorklog(Stats):
+    """ Jira Issues for which a worklog entry was made """
+
+    def fetch(self):
+        log.info(
+            "[%s] Searching for issues for which work was logged by '%s'",
+            self.option,
+            self.user.login or self.user.email)
+        query = (
+            f"worklogAuthor = '{self.user.login or self.user.email}' "
+            f"and worklogDate >= {self.options.since} "
+            f"and workLogDate < {self.options.until} "
+            )
+        if self.parent.project:
+            query = query + f" AND project in ({self.parent.project})"
+        issues = Issue.search(query, stats=self, with_worklog=True)
+        # Now we have just the issues which have work logs but we
+        # want to limit what worklogs we include in the report.
+        # Filter out worklogs that were not done in the given
+        # time frame.
+        log.debug("Found issues: %d", len(issues))
+        for issue in issues:
+            log.debug("Found worklogs: %s", len(issue.worklogs))
+            issue.worklogs = [wl for wl in issue.worklogs if
+                              (("name" in wl["author"]
+                                and wl["author"]["name"] == self.user.login)
+                               or ("emailAddress" in wl["author"]
+                                   and wl["author"]["emailAddress"] == self.user.email))
+                              and dateutil.parser.parse(wl["created"]).date()
+                              >= self.options.since.date
+                              and dateutil.parser.parse(wl["created"]).date()
```

Comment:
As I expect this date parsing will be done often if worklog is used, perhaps better parse `created` date once and store it in a variable. Or use a chained comparison like:
```python
self.options.since.date <= dateutil.parser.parse(wl["created"]).date() < self.options.until.date
```

So the parsing will be done only once.